### ISSUE TITLE
Native notifications are HTML-encoded

### ISSUE THREAD

**Issue by Defman21** [telegramdesktop/tdesktop#2532]
### Steps to reproduce
1. Enable native notifications (on Linux)
1. Ask an user to send you `<>`.

### Expected behavior
A notification with `<>` as the message.

### Actual behavior
A notification with `&lt;&gt;` as the message.

### Configuration
**Operating system:** elementary OS 0.4

**Version of Telegram Desktop:** 0.10.13 alpha

### Note

It works fine if you run `notify-send "<>"`.

**Comment by Defman21** [comment]
I don't know libnotify specs so I don't understand your answer. Have you confirmed that the report is a bug?

**Comment by Defman21** [comment]
So it's a bug in the distro and how it displays notifications, and not in the Telegram app?

**Comment by john-preston** [comment]
@Defman21 what does it show if you try:

notify-send "<b>test</b>"

?

**Comment by Defman21** [comment]
Bold text like in html

**Comment by Defman21** [comment]
![image](https://cloud.githubusercontent.com/assets/7100645/19526993/6800a5f0-9637-11e6-8e4a-fdc439b471c0.png)

**Comment by john-preston** [comment]
Then if you try:

    notify-send "&lt;b&gt;test&lt;/b&gt;"

what does it show?

It should show a notification with "<b>test</b>" text, not with "&lt;b&gt;test&lt;/b&gt;" text, right? Or how would you show such text in the notification at all?

**Comment by Defman21** [comment]
My first image was incorrect because title is always bold, so here's the right one:

![image](https://cloud.githubusercontent.com/assets/7100645/19527468/2f90fac4-9639-11e6-9f3d-58ba649d4722.png)

With the second command, I get this notification.

![image](https://cloud.githubusercontent.com/assets/7100645/19527456/236ecf50-9639-11e6-87f9-a7d1d3dfb179.png)

**Comment by john-preston** [comment]
Well.. So notify-send converts entities? To be sure, those both commands give you the same result?

    notify-send "<>"
    notify-send "&lt;&gt;"

**Comment by Defman21** [comment]
Yes :)

**Comment by john-preston** [comment]
Then I don't know why my call to notify_notification_new() does not :( Because if I won't convert "&lt;" to "&amp;lt;" it will show a bold string for a message like "&lt;b&gt;test&lt;/b&gt;".

**Comment by Defman21** [comment]
Are you sure that notify-send and notify_notification_new() behaves in the same way? Maybe if you'll send `<b>Text</b>` via `notify_notification_new()`, it won't be parsed as HTML.

**Comment by john-preston** [comment]
@Defman21 looks like I've found the problem. I escape the text no matter the "body-markup" capability %) Can you please look at log.txt (it should be at ~/.TelegramDesktop/log.txt or ~/.local/share/TelegramDesktop/log.txt) and copy a string containing "LibNotify capabilities:" here?

**Comment by Defman21** [comment]
Yeah no problem, I'll post it when I'll be at home 🙂

**Comment by Defman21** [comment]
```
LibNotify capabilities: body, body-markup, sound, actions, x-canonical-private-synchronous, x-canonical-private-icon-only
```

**Comment by john-preston** [comment]
Well, then I didn't find the problem: libnotify ensures me that it supports body-markup, so it displays &lt;b&gt;test&lt/b&gt; as <b>test</b> - so I must encode entities before I send text to it :(

**Comment by Defman21** [comment]
Looks like you're escaping escaped characters.

This code gives me normal output.

```
#include <libnotify/notify.h>

int main() {
	notify_init ("Hello world!");
	NotifyNotification *hello = notify_notification_new ("Hello world", "This is an example notification. <b>World</b> &lt;b&gt;bold&lt;/b&gt;", "dialog-information");
	notify_notification_show (hello, NULL);
	g_object_unref(G_OBJECT(hello));
	notify_uninit();
	return 0;
}
```

![image](https://cloud.githubusercontent.com/assets/7100645/19566810/a60b0bd6-96fc-11e6-9304-49828e94d62a.png)

**Comment by Defman21** [comment]
This code gives me the same notification as Telegram does.

```
#include <libnotify/notify.h>

int main() {
	notify_init ("Hello world!");
	NotifyNotification *hello = notify_notification_new ("Hello world", "This is an example notification. &amp;lt;b&amp;gt;bold&amp;lt;/b&amp;gt;", "dialog-information");
	notify_notification_show (hello, NULL);
	g_object_unref(G_OBJECT(hello));
	notify_uninit();
	return 0;
}
```

![image](https://cloud.githubusercontent.com/assets/7100645/19566911/08784252-96fd-11e6-98f7-9d49431ad43c.png)

**Comment by john-preston** [comment]
Thanks, I'll have a look.

**Comment by john-preston** [comment]
I've found the problem, thanks for your help.

**Comment by Defman21** [comment]
0.10.19 - the issue is fixed. Thank you, @john-preston! :)
